components/ConjugationPage/ConjugationPage.js

```diff
@@ -0,0 +1,89 @@
+import styled from "styled-components";
+import { useState } from "react";
+import ConjugationButtons from "../ConjugationButtons/ConjugationButtons";
+
+export default function ConjugationPage({ currentWord }) {
+  const present = currentWord.query1.conjugation.present;
+  const past = currentWord.query1.conjugation.past;
+  const future = currentWord.query1.conjugation.future;
+  const [tense, setTense] = useState("present");
+
+  function changeTense(tense) {
+    setTense(tense);
+  }
+
+  return (
+    <>
+      <StyledSection>
+        <ConjugationButtons tense={tense} onChangeTense={changeTense} />
+        <StyledTable>
+          <tbody>
+            {tense === "present" &&
+              present.map((word) => (
+                <StyledTableRowsPresent key={word.person}>
+                  <StyledPronouns>{word.pronouns}</StyledPronouns>
+                  <StyledVerbform>{word.verbForm}</StyledVerbform>
+                </StyledTableRowsPresent>
+              ))}
+            {tense === "past" &&
+              past.map((word) => (
+                <StyledTableRows key={word.person}>
+                  <StyledPronouns>{word.pronouns}</StyledPronouns>
+                  <StyledVerbform>{word.verbForm}</StyledVerbform>
+                </StyledTableRows>
+              ))}
+            {tense === "future" &&
+              future.map((word) => (
+                <StyledTableRows key={word.person}>
+                  <StyledPronouns>{word.pronouns}</StyledPronouns>
+                  <StyledVerbform>{word.verbForm}</StyledVerbform>
+                </StyledTableRows>
+              ))}
+          </tbody>
+        </StyledTable>
+      </StyledSection>
+    </>
+  );
+}
+
+const StyledTable = styled.table`
+  padding: 0.625rem;
```

Comment:
okay, THAT's a pretty specific measurement :D

Reply:
I had pixels before - just calculated from there ;)